src/components/workspace/side-content/AutograderCard.tsx

```diff
@@ -33,19 +33,19 @@ class AutograderCard extends React.Component<AutograderCardProps, {}> {
             )}
           </div>
           <div className="row autograder-program">
-            <pre className="code">{this.props.testcase.program}</pre>
+            <Pre className="bp3-code-block">{this.props.testcase.program}</Pre>
```

Comment:
Please revert this change. `.code` is a class, `.bp3-code-block` applies to pre elements that were previously selected in bp2.